src/CoreLocation/CLBeaconRegion.cs

```diff
@@ -8,7 +8,13 @@ namespace CoreLocation {
 
 	public partial class CLBeaconRegion {
 
+#if NET
```

Comment:
Friendly reminder - these come from the [definition attributes](https://github.com/xamarin/xamarin-macios/blob/main/src/corelocation.cs#L721-L722).